packages/terra-slide-panel/src/terra-dev-site/doc/example/DefaultSlidePanel.jsx

```diff
@@ -133,7 +133,7 @@ class DefaultSlidePanel extends React.Component {
 
 DefaultSlidePanel.propTypes = {
   panelBehavior: PropTypes.oneOf(['overlay', 'squish']),
-  panelPosition: PropTypes.oneOf(['start', 'end']),
+  panelPosition: PropTypes.oneOf([SlidePanel.position.START, SlidePanel.position.END]),
```

Comment:
Lets use the strings in the example.

Reply:
Reverted the changes to use strings instead of enums